salt/hubblestack/pulsar_dependency.sls

```diff
@@ -0,0 +1,3 @@
+install_pulsar_dependency:
+  pkg.installed:
+    - name: python-pyinotify
```

Comment:
We have a state file in `salt/utils/install_pip.sls` that allows for installation of arbitrary python packages driven by pillar data in the form
```yaml
python_dependencies:
    python_libs:
        - pyinotify
        - someotherpackage
```
I believe that state is already included in the `top.sls` for all of the minions.